app/classes/Controllers/ComponentDetail.php

```diff
@@ -66,6 +66,7 @@ public function get() {
 		$this->addViewData('dependents', $this->version->dependents);
 
 		$this->addViewData('title', $this->component->module_name);
+		$this->addViewData('keywords', $this->component->latest_version->keywords);
```

Comment:
I don't think this is needed. The keywords aren't output in the view detail view.